husky_components/column-navigation/main.js

```diff
@@ -1344,7 +1344,9 @@ define(function() {
          */
         mark: function(id, item) {
             var $element = this.$find('li[data-id="' + id + '"]');
-            if (!!$element.length) {
+
+            // mark only non-ghost pages if showActionButtonOnGhost is disabled
+            if (!!$element.length && ((!item.type || !!item.type && item.type.name !== 'ghost') || this.options.showActionButtonOnGhost)) {
```

Comment:
I think we can get rid of `!!item.type &&` in that case. If `!item.type` is not true, then `item.type` exists and the other statement will work as expected.